packages/jazz-tools/src/coValues/coList.ts

```diff
@@ -221,7 +221,7 @@ export class CoList<Item = any> extends Array<Item> implements CoValue {
   static create<L extends CoList>(
     this: CoValueClass<L>,
     items: UnCo<L[number]>[],
-    options: { owner: Account | Group } | Account | Group,
+    options?: { owner: Account | Group } | Account | Group,
```

Comment:
Do we want this to default to group when no value is passed?

@aeplay

Reply:
It's a bit of a bold move, but yes - because it's more fixable later